test/Tests/Strings/DiscreteCharTest.cs

```diff
@@ -64,6 +64,31 @@ public void BadRanges()
             Assert.True(false);
         }
 
+        [Fact]
+        [Trait("Category", "StringInference")]
+        public void SampleFromUniformCharDistribution()
+        {
```

Comment:
Usual want to do a Rand.Restart to have a deterministic test.

Reply:
Added a Restart()